securityconfig/elasticsearch.yml.example

```diff
@@ -215,11 +215,11 @@ opendistro_security.audit.type: internal_elasticsearch
 # Specify a list of roles a user must be member of to touch any protected index.
 # opendistro_security.protected_indices.roles: ['all_access']
 # Specify a list of indices to mark as protected. These indices will only be visible / mutable by members of the above setting, in addition to needing permission to the index via a normal role.
-# opendistro_security.protected_indices.indices: ['.opendistro-alerting-config', '.opendistro-ism-*']
+# opendistro_security.protected_indices.indices: ['.opendistro-alerting-config', '.opendistro-ism-*', '.opendistro-reports-*', '.opendistro-notifications-*']
```

Comment:
This is example, but curious why you want to add reports to protected_indices?

Reply:
If system index is not enabled then could it be configured as protected?

Reply:
are these indices created only for plugin to consume (like alerting and ISM indices )? 
does any of the users including admin need direct access to this indices using ES APIs?

Reply:
are these indices created only for plugin to consume (like alerting and ISM indices )? --> YES
does any of the users including admin need direct access to this indices using ES APIs? --> NO. users should not have direct access

Reply:
then these indices should be part of system indices and not protected indices. 

protected indices will be deprecated once all plugin move their indices to system indices.

Reply:
Yes, protected_indices will be deprecated.

Reply:
will remove.